specs/darwin/alf_services.table

```diff
@@ -1,8 +1,15 @@
 table_name("alf_services")
 description("OS X application layer firewall (Firewall) services.")
 schema([
-    Column("service", TEXT, "Firewalled service name"),
-    Column("process", TEXT, "Process name"),
-    Column("state", INTEGER, "Firewall service state"),
```

Comment:
I'd suggest to keep this columns, but mark them as deprecated until the next major release. It's not fair to change interface significantly without major version update. What do you think?

Reply:
Sounds good! Should I stub them out with default values, or leave them in as-is?

Reply:
I didn't think about it, but there are some folks who have queries based on that columns. Why do we need to remove them? Does table only with numbers in columns make some sense?

Reply:
> Why do we need to remove them?

They don't reflect the actual firewall state. Here's what `alf_services` looks like when I have Apple Events and Printer Sharing **enabled**:

```
osquery> select * from alf_services;
+-----------------------+---------------------+-------+
| service               | process             | state |
+-----------------------+---------------------+-------+
| Personal Web Sharing  | httpd               | 0     |
| Printer Sharing       | cupsd               | 0     |
| Remote Apple Events   | AEServer            | 0     |
| FTP Access            | ftpd                | 0     |
| Personal File Sharing | AppleFileServer     | 0     |
| Remote Login - SSH    | sshd-keygen-wrapper | 0     |
| Samba Sharing         | smbd                | 0     |
| Apple Remote Desktop  | AppleVNCServer      | 0     |
| ODSAgent              | ODSAgent            | 0     |
+-----------------------+---------------------+-------+
```

what I **expect** is this:

```
osquery> select * from alf_services;
+-----------------------+---------------------+-------+
| service               | process             | state |
+-----------------------+---------------------+-------+
| Personal Web Sharing  | httpd               | 0     |
| Printer Sharing       | cupsd               | 1     |
| Remote Apple Events   | AEServer            | 1     |
| FTP Access            | ftpd                | 0     |
| Personal File Sharing | AppleFileServer     | 0     |
| Remote Login - SSH    | sshd-keygen-wrapper | 0     |
| Samba Sharing         | smbd                | 0     |
| Apple Remote Desktop  | AppleVNCServer      | 0     |
| ODSAgent              | ODSAgent            | 0     |
+-----------------------+---------------------+-------+
```

Essentially, the underlying store (`firewall` in `com.apple.alf.plist`) is no longer used by macOS.  `sharing_preferences` is the only table (AFAIK) that accurately reflects service state.